src/pages/[teamName]/[projectName]/configuration.tsx

```diff
@@ -587,13 +592,46 @@ const ConfigurationPage = withError<
                 />
               </FormControl>
 
+              <FormControl d="flex" alignItems="center" my={4}>
```

Comment:
Is it possible to roll this into a loop and extract out a component, as there is some code duplication here.